salt/modules/network.py

```diff
@@ -1309,6 +1331,8 @@ def default_route(family=None):
     for route in _routes:
         if family:
             if route['destination'] in default_route[family]:
+                if __grains__['kernel'] == 'SunOS' and route['addr_family'] != family:
```

Comment:
@sjorge, is there a reason the second condition is predicated upon the platform being `SunOS`?